packages/network/bluez/package.mk

```diff
@@ -35,11 +35,7 @@ else
   BLUEZ_CONFIG="--disable-debug"
 fi
 
-if [ "$DEVTOOLS" = "yes" ]; then
-  BLUEZ_CONFIG="$BLUEZ_CONFIG --enable-monitor --enable-test"
-else
-  BLUEZ_CONFIG="$BLUEZ_CONFIG --disable-monitor --disable-test"
-fi
+BLUEZ_CONFIG="$BLUEZ_CONFIG --enable-monitor --enable-test"
```

Comment:
` --enable-test` do we need the testscripts at all ?

Reply:
no idea, but they are only 224k uncompressed and compress down to ~28k
```
hias@tv:~/rpi/libreelec-master$ du -s -h build.LibreELEC-RPi2.arm-9.0-devel/image/system/usr/lib/bluez/test
224K    build.LibreELEC-RPi2.arm-9.0-devel/image/system/usr/lib/bluez/test
hias@tv:~/rpi/libreelec-master$ tar zcf /tmp/test.tgz build.LibreELEC-RPi2.arm-9.0-devel/image/system/usr/lib/bluez/test
hias@tv:~/rpi/libreelec-master$ ls -lh /tmp/test.tgz
-rw-r--r-- 1 hias hias 28K Apr 25 19:32 /tmp/test.tgz
```

Reply:
@lrusak might know !?